entity-types/infra-apacheserver/opentelemetry_dashboard.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "Open Telemetry Apache Server",
+  "pages": [
+    {
+      "name": "Open Telemetry Apache Server",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.line"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 6
+          },
+          "title": "Requests per second",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "query": "SELECT average(apache.requests) AS `Requests` FROM Metric WHERE otel.library.name='otelcol/apachereceiver' TIMESERIES AUTO",
+                "accountId": 0}
+            ]
+          }
+        },
+        {
+          "visualization": {
+            "id": "viz.line"
+          },
+          "layout": {
+            "column": 7,
+            "row": 1,
+            "height": 3,
+            "width": 6
+          },
+          "title": "Total HTTP server traffic (Bytes)",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "query": "FROM Metric SELECT average(apache.traffic) WHERE otel.library.name = 'otelcol/apachereceiver' TIMESERIES AUTO",
+                "accountId": 0}
+            ]
+          }
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 1,
+            "row": 4,
+            "height": 3,
+            "width": 12
+          },
+          "title": "Worker Status",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "query": "FROM Metric SELECT latest(apache.scoreboard) FACET upper(state)",
```

Comment:
Is there any reason why including the filter `WHERE otel.library.name='otelcol/apachereceiver'` in all the dashboard's queries except this one?

Reply:
Thats a miss. Thanks for catching that. 
Updated the query